packages/components/package.json

```diff
@@ -133,7 +133,7 @@
     "semantic-release": "^19.0.5",
     "semantic-release-monorepo": "^7.0.5",
     "sinon": "^17.0.0",
-    "storybook": "^7.3.2",
+    "storybook": "^7.5.1",
```

Comment:
Will this be an issue?? 😅

Reply:
@yoezlem

Reply:
What do you mean?
If you mean to separate it, you don't have to do it in this PR. Just keep in mind to do it next time.